DFM/dfmSession.c

```diff
@@ -182,12 +183,13 @@ DfmResult_t xDfmSessionDisable(uint32_t ulRemember)
 	if (ulStoredEnabledValue != DFM_DISABLED)
 	{
 		/* We didn't find disabled */
-		((DfmSessionStorage_t*)cSessionStorageBuffer)->ulVersion = DFM_SESSION_STORAGE_VERSION; /*cstat !MISRAC2012-Rule-11.3 We use an untyped buffer to retrieve the session data since we can't know what version and size it might have been stored in the past. The stored Session data might be larger than the current DfmSessionStorage_t*/
-		((DfmSessionStorage_t*)cSessionStorageBuffer)->ulEnabled = DFM_DISABLED; /*cstat !MISRAC2012-Rule-11.3 We use an untyped buffer to retrieve the session data since we can't know what version and size it might have been stored in the past. The stored Session data might be larger than the current DfmSessionStorage_t*/
+		DfmSessionStorage_t *pSessionStorage = (DfmSessionStorage_t*)&cSessionStorageBuffer[0];
```

Comment:
Please rename this to pxSessionStorage instead.

Reply:
Fixed